packages/node_modules/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -4,9 +4,70 @@
 
 import {WebexPlugin} from '@webex/webex-core';
 
+import CalendarCollection from './collection';
+
 const Calendar = WebexPlugin.extend({
   namespace: 'Calendar',
 
+  register() {
```

Comment:
Could you add device.refresh and mercury.connect here and move the listen events to a `listenToEvents` function like the meetings plugin?

Reply:
yes